src/main/features/offer/views/settled.njk

```diff
@@ -7,28 +7,28 @@
   <div class="grid-row">
     <div class="column-two-thirds">
       <div class="govuk-box-highlight">
-        <h1 class="heading-large">{{ t('You’ve settled out of court') }}</h1>
-
-        <p class="issue-receipt-download">
-          <a href="{{ agreementLink }}">{{ t('Download agreement as a PDF') }}</a>
-        </p>
+        <h1 class="bold-large">{{ t('You’ve settled out of court') }}</h1>
+        <div class="action-date-container">
+          <strong class="bold">{{ claim.settlementReachedAt | date }}</strong>
```

Comment:
no need for strong. this should just be a span/div/p or if using strong then there should be no need for the bold class

Reply:
Valid point, will change it.

Reply:
Done